src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,56 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1_3
+    def test_0019_bz2070189(self, multihost, adjoin):
+        """
+        :title: sssd error triggers backtrace
+        :description:
+            Global Catalog lookups should not update the kdcinfo files
+        :id: 91bd8845-20e4-4fde-9914-73b08d44c195
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2070189
+        :steps:
+            1. setup AD environment
+            2. Join linux machine into AD domain
+            3. Set the debug_level in sssd.conf for AD domain to 1,
+               make sure backtrace dump is enabled,
+               debug_backtrace_enabled=True
+               make sure that krb5_use_kdcinfo=True
+            4. clear the cache sss_cache -E
+            5. lookup AD user `id administrator@domain`
+        :expectedresults:
+            1. null
+            2. null
+            3. null
+            4. null
+            5. Backtrace is NOT triggered
+        """
+        adjoin(membersw='adcli')
+        backup = 'cp -f /etc/sssd/sssd.conf /etc/sssd/sssd.conf.orig'
+        restore = 'cp -f /etc/sssd/sssd.conf.orig /etc/sssd/sssd.conf'
+        multihost.client[0].run_command(backup)
```

Comment:
client = sssdTools(multihost.client[0], multihost.ad[0])
        client.backup_sssd_conf()

Reply:
We got function for backup already (also a fixture for backup is available).